### ISSUE TITLE
Unable to compile with Apktool

### ISSUE THREAD

**Issue by Magnusretrotech** [ftde0/yt2009#118]
I followed the APK editing guide with YouTube 1.6.20, but when I tried to recompile it, it gave this output and error.

`I: Using Apktool 2.9.3
I: Checking whether sources has changed...
I: Checking whether resources has changed...
I: Building resources...
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:9: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:10: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:11: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:12: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:13: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:14: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:15: error: invalid value for type 'attr'. Expected a reference.
W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml: error: file failed to compile.
brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_90055360671254054293145669519737815366.tmp, compile, --dir, /home/magnus/Downloads/old youtube/youtube/res, --legacy, -o, /home/magnus/Downloads/old youtube/youtube/build/resources.zip]`

Any help would be greatly appreciated

**Comment by Magnusretrotech** [comment]
Oh I forgot to mention I'm using Fedora linux

**Comment by Magnusretrotech** [comment]
I have also tried using autoapk (which was completely broken btw, I had to manually install the dependencies and then rewrite about 30% of the whole script, but it ran into the exact same error (when it tried to recompile it).

**Comment by gigigigi53** [comment]
> I followed the APK editing guide with YouTube 1.6.20, but when I tried to recompile it, it gave this output and error.
> 
> `I: Using Apktool 2.9.3 I: Checking whether sources has changed... I: Checking whether resources has changed... I: Building resources... W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:9: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:10: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:11: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:12: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:13: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:14: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:15: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml: error: file failed to compile. brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_90055360671254054293145669519737815366.tmp, compile, --dir, /home/magnus/Downloads/old youtube/youtube/res, --legacy, -o, /home/magnus/Downloads/old youtube/youtube/build/resources.zip]`
> 
> Any help would be greatly appreciated

Not sure why attr is invalid value! The same for 1.3, 1.4.17, 1.4.20, 1.5.20 and 1.6.21.

**Comment by Magnusretrotech** [comment]
> > I followed the APK editing guide with YouTube 1.6.20, but when I tried to recompile it, it gave this output and error.
> > `I: Using Apktool 2.9.3 I: Checking whether sources has changed... I: Checking whether resources has changed... I: Building resources... W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:9: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:10: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:11: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:12: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:13: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:14: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml:15: error: invalid value for type 'attr'. Expected a reference. W: /home/magnus/Downloads/old youtube/youtube/res/values/attrs.xml: error: file failed to compile. brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_90055360671254054293145669519737815366.tmp, compile, --dir, /home/magnus/Downloads/old youtube/youtube/res, --legacy, -o, /home/magnus/Downloads/old youtube/youtube/build/resources.zip]`
> > Any help would be greatly appreciated
> 
> Not sure why attr is invalid value! The same for 1.3, 1.4.17, 1.4.20, 1.5.20 and 1.6.21.

Yeah, I sadly can't use any newer versions since I'm running this on android 2.1
